projectFilesBackup/.idea/workspace.xml

```diff
@@ -0,0 +1,477 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="AndroidLayouts">
+    <shared>
+      <config />
+    </shared>
+  </component>
+  <component name="AndroidLogFilters">
+    <option name="TOOL_WINDOW_CONFIGURED_FILTER" value="Show only selected application" />
+  </component>
+  <component name="ChangeListManager">
+    <list default="true" id="b395262b-9905-4ad3-96a5-857dfa73d10c" name="Default Changelist" comment="">
+      <change afterPath="$PROJECT_DIR$/app/src/main/java/com/github/sdpteam15/polyevents/helper/GoogleMapAdapter.kt" afterDir="false" />
```

Comment:
Is this file really needed on the online repo ? :)

Reply:
Nope, it has also been removed by Mathieu :)